extras/docker/qns/run_endpoint.sh

```diff
@@ -49,10 +47,8 @@ check_testcase () {
 run_quiche_client_tests () {
     for req in $REQUESTS
```

Comment:
No need to use `for`. Can you apply my patch https://github.com/cloudflare/quiche/pull/331#issuecomment-579557222

Reply:
oops, will fix that tomorrow